README.md

```diff
@@ -85,15 +85,25 @@ System OpenSSL library is required.
   - set INDY_CRYPTO_PREBUILT_DEPS_DIR=C:\BIN\x64
   - set OPENSSL_DIR=C:\BIN\x64
 
-## API Documentation
+## Documentation
+Project documentation is available via sphinx.
+Install sphinx:
+```
+pip install -U Sphinx
```

Comment:
Why do we need sphinx?
Is this your preferred documentation generator?

Reply:
Yep. It's what we use for https://sawtooth.hyperledger.org/docs/

Reply:
And sphinx generates the Makefile. I wouldn't see us using Makefiles broadly in this project.

Reply:
okay as long as we are not committing to Makefiles for now I'm okay with it.